gensim/models/word2vec.py

```diff
@@ -190,7 +190,11 @@ def train_batch_sg(model, sentences, alpha, work=None, compute_loss=False):
 
                 # now go over all words from the (reduced) window, predicting each one in turn
                 start = max(0, pos - model.window + reduced_window)
-                for pos2, word2 in enumerate(word_vocabs[start:(pos + model.window + 1 - reduced_window)], start):
+                if not hasattr(model, 'symmetric') or model.symmetric:
```

Comment:
part with `hasattr` should be in `load` function and initialize it with 1 as default value